### ISSUE TITLE
Permission problems with non sudoers users on self-hosted runners

### ISSUE THREAD

**Issue by MrFastwind** [CyberAndrii/setup-steamcmd#4]
if the user of the runner isn't in sudoers, it can't run installation of libraries; this make difficult working with self-hosted runners.
a possible solution would be to check the presence of the package before trying installing it.

platform: self-hosted [linux]

**Comment by andriivitiv** [comment]
Any reason the user is not in sudoers?

You need to add the user to `sudo` group (e.g. using command `usermod -aG sudo <username>`) and remove password (e.g. using `echo '%sudo ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers`). The easyest way is to use https://github.com/myoung34/docker-github-actions-runner (or similar) which do so automatically for you.

This doesn't give/require the root access so there's nothing to worry about. **UPD: wrong.**

**Comment by MrFastwind** [comment]
The runner is hosted directly on the machine, but the action i run on top of it could come from outsiders, so i can't give it sudo permision, because i can't descriminate which action could run as root.
Also i have permission to install dependencies, that why i suggested to check if is installed before trying.
Also i can't run it on docker.

**Comment by andriivitiv** [comment]
Pushed https://github.com/CyberAndrii/setup-steamcmd/commit/78c93f37837a8eb7af73a12be29a7b5472455533 . Let me know if this works for you. Use it with
```yml
uses: CyberAndrii/setup-steamcmd@78c93f37837a8eb7af73a12be29a7b5472455533
```

**Comment by MrFastwind** [comment]
Yes it does work, thanks 👍

**Comment by andriivitiv** [comment]
Released v1.1.4
